File: solvings/easy/running-sum-of-1d-array.cs
public class Solution
{
    public int[] RunningSum(int[] nums)
    {
        var sum = nums[0];
        var result = new int[nums.Length];
        result[0] = sum;

        for (int i = 1; i < nums.Length; i++)
        {
            sum += nums[i];
            result[i] = sum;
        }
        return result;
    }
}

File: solvings/easy/running-sum-of-1d-array.go
package solvings

func runningSum(nums []int) []int {
	result := make([]int, len(nums))
	sum := 0
	for idx := 0; idx < len(nums); idx++ {
		sum += nums[idx]
		result[idx] = sum
	}

	return result
}
